NGINX Plus Release 34 (#343)
nginx/documentation: content/nginx/deployment-guides/single-sign-on/oidc-njs/keycloak.md

> ---
> description: Enable OpenID Connect-based single-sign for applications proxied by NGINX
>   Plus, using Keycloak as the identity provider (IdP).
> docs: DOCS-465
> doctypes:
> - task
> title: Single Sign-On with Keycloak and njs
> toc: false
> weight: 100
> ---

Reviewer: ```suggestion
---
description: Enable OpenID Connect-based single-sign for applications proxied by NGINX
  Plus, using Keycloak as the identity provider (IdP).
docs: DOCS-465
type:
- how-to
title: Single Sign-On with Keycloak and njs
toc: false
weight: 100
product: NGINX-PLUS
---
```
